File: 3299 - Números Má Sorte Pequenos.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

if(lines[0].includes("13")) console.log(`${lines[0]} es de Mala Suerte`)
else console.log(`${lines[0]} NO es de Mala Suerte`)

File: 3299 - Números Má Sorte Pequenos.cs
using System;

public class Program
{
    public static void Main()
    {
        string n = Console.ReadLine();
        if (n.Contains("13")) Console.WriteLine($"{n} es de Mala Suerte");
        else Console.WriteLine($"{n} NO es de Mala Suerte");
    }
}
